### PR TITLE
add node delete

### PR DESCRIPTION
**Pull request by pyToshka**
Hey, I'm add node delete mechanism for library

### CODE REVIEW

File: jenkins.go [hunk 1]
```diff
@@ -168,6 +168,19 @@ func (j *Jenkins) CreateNode(name string, numExecutors int, description string,
 	}
 	return nil, errors.New(strconv.Itoa(resp.StatusCode))
 }
+//delete node
```
Comment (supereagle) [inline jenkins.go]: Please follow the style of other comments, like capitalize, space and so on.

File: jenkins.go [hunk 2]
```diff
@@ -168,6 +168,19 @@ func (j *Jenkins) CreateNode(name string, numExecutors int, description string,
...
+// deregister jenkins slave
+//example
+//node_del,_:=jenkins.DeleteNode(id1)
+//fmt.Sprintf("%s",node_del)
```
Comment (supereagle) [inline jenkins.go]: This line is not needed.

File: jenkins.go [hunk 3]
```diff
@@ -168,6 +168,19 @@ func (j *Jenkins) CreateNode(name string, numExecutors int, description string,
...
+func (j *Jenkins) DeleteNode(name string) (bool, error) {
+	node := Node{Jenkins: j, Raw: new(NodeResponse), Base: "/computer/" + name}
+	delete,err :=node.Delete()
+	if err != nil {
+		errors.New("Error Deleting node")
+	}
+	return delete,err
+	}
```
Comment (supereagle) [inline jenkins.go]: go fmt

File: jenkins.go [hunk 4]
```diff
@@ -168,6 +168,19 @@ func (j *Jenkins) CreateNode(name string, numExecutors int, description string,
...
 }
+//delete node
+// deregister jenkins slave
+//example
+//node_del,_:=jenkins.DeleteNode(id1)
+//fmt.Sprintf("%s",node_del)
+func (j *Jenkins) DeleteNode(name string) (bool, error) {
+	node := Node{Jenkins: j, Raw: new(NodeResponse), Base: "/computer/" + name}
+	delete,err :=node.Delete()
+	if err != nil {
```
Comment (supereagle) [inline jenkins.go]: No need to check the err, can directly return `node.Delete()`.

**Comment by pyToshka** [comment]
Please take a look.

**Pull request closed without merging**